.travis/build

```diff
@@ -18,6 +18,7 @@ apt-get install -qq -y libignition-msgs4-dev \
                        python-rosdep \
                        ros-melodic-geometry-msgs \
                        ros-melodic-mav-msgs \
+                       ros-melodic-nav-msgs \
                        ros-melodic-rostest \
                        ros-melodic-sensor-msgs
```

Comment:
Do we need to list the ROS deps here even though we're using `rosdep install` below?

Reply:
> Do we need to list the ROS deps here even though we're using `rosdep install` below?

I don't think so.  ced631f